scripts/s_aureus/generate_results.py

```diff
@@ -123,93 +121,60 @@ def build_csv_list(sample):
 
 
 @click.command()
-@click.option(
-    "--metadata-file",
-    type=click.Path(exists=True, file_okay=True, readable=True),
-    required=True,
-    help="the sample metadata file",
-)
-@click.option(
-    "--output-csv-file",
-    type=click.Path(dir_okay=False, writable=True),
-    required=True,
-    help="output file merging the results from ncov and pangolin",
-)
-@click.option(
-    "--output-json-file",
-    type=click.Path(dir_okay=False, writable=True),
-    required=True,
-    help="output file containing all the expected files per sample",
-)
 @click.option(
     "--output-path",
     type=str,
     required=True,
     help="output_path output path where sample result files are stored (e.g. s3://bucket/path/analysis_run)",
 )
-@click.option(
-    "--sequencing-technology",
-    type=click.Choice([ILLUMINA, ONT], case_sensitive=True),
-    required=True,
-    help="the sequencer technology used for sequencing the samples",
-)
 def generate_results(
-    metadata_file: str,
-    output_csv_file: str,
-    output_json_file: str,
     output_path: str,
-    sequencing_technology: str,
 ) -> None:
     """
     Generate pipeline results files
     """
-    print('metadata_file: ' + metadata_file)
-    print('output_csv_file: ' + output_csv_file)
-    print('output_json_file: ' + output_json_file)
-    print('output_path: ' + output_path)
-    print('sequencing_technology: ' + sequencing_technology)
-
-    # Make a list so we can iterate over this several times easily
-    # TODO change to single sample -- get sample another way
-    metadata = list(csv.DictReader(open(Path(metadata_file))))
-
-    sample = metadata[0]
+    # Get sample_id and genome_assembly_name
+    with open('genome_assembly_name.txt') as genome_assembly_name_file:
```

Comment:
This should be a `.json` not `.txt`. It is not obvious at all.

Reply:
Agreed, wont fix because deadline is today